src/App.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './App.css';
+import {Page} from "./Page";
+import {BrowserRouter} from "react-router-dom";
+import Menu from "./Menu";
+import Switch from "react-router-dom/es/Switch";
+import Route from "react-router-dom/es/Route";
+import Redirect from "react-router-dom/es/Redirect";
+
+function App() {
+    return (
+            <div className="App">
+                <BrowserRouter>
+                    <Switch>
+                        <Route path={"/"+Menu.PAGE_HOME}
```

Comment:
This is kinda weird. What page exactly should be displayed should be chosen by router, not inside of some component based on ifs.